catalog/model/extension/module/altapay.php

```diff
@@ -32,4 +32,28 @@ public function saveOrderReconciliationIdentifier($order_id, $reconciliation_ide
             }
         }
     }
+
+    /**
+     * @param int $order_id
+     * @param bool $capture
+     * @param bool $refund
+     * @param bool $void
+     *
+     * @return void
+     */
+    public function updateOrderMeta($order_id, $capture = false, $refund = false, $void = false) {
+        if (filter_var($order_id, FILTER_VALIDATE_INT, ['options' => ['min_range' => 1]]) === false) {
```

Comment:
Is it correct that you allow the updateOrderMeta to finish without storing the values, neither failing the execution?

Reply:
Yes, that is correct.